### PR TITLE
add IMAGE_PROCESSING

### PR DESCRIPTION
**Pull request by dstrande**
Includes docs for IMAGE_SMOOTHING and EDGE_DETECTION.

### CODE REVIEW

File: docs/nodes/TRANSFORMERS/IMAGE_PROCESSING/EDGE_DETECTION/a1-[autogen]/python_code.txt [hunk 1]
```diff
@@ -0,0 +1,48 @@
+from flojoy import flojoy, DataContainer
+import cv2
+import numpy as np
+
+
+@flojoy
+def IMAGE_SMOOTHING(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
```
Comment (smahmed776) [inline docs/nodes/TRANSFORMERS/IMAGE_PROCESSING/EDGE_DETECTION/a1-[autogen]/python_code.txt:7]: I think you put wrong code file here

File: docs/nodes/TRANSFORMERS/IMAGE_PROCESSING/EDGE_DETECTION/a1-[autogen]/parameters.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+kernel:
```
Comment (smahmed776) [inline docs/nodes/TRANSFORMERS/IMAGE_PROCESSING/EDGE_DETECTION/a1-[autogen]/parameters.yaml:1]: I think this was by mistake. As I can see EDGE_DETECTION doesn't accept any parameters

File: docs/nodes/TRANSFORMERS/IMAGE_PROCESSING/EDGE_DETECTION/a1-[autogen]/docstring.txt [hunk 3]
```diff
@@ -0,0 +1,22 @@
+
+    Apply image smoothing operation on the input `DataContainer` class,
```
Comment (smahmed776) [inline docs/nodes/TRANSFORMERS/IMAGE_PROCESSING/EDGE_DETECTION/a1-[autogen]/docstring.txt:2]: I think this docstring is related to to Image smoothing not edge detection

Comment (dstrande) [reply docs/nodes/TRANSFORMERS/IMAGE_PROCESSING/EDGE_DETECTION/a1-[autogen]/docstring.txt]: Fixed

File: docs/nodes/TRANSFORMERS/IMAGE_PROCESSING/EDGE_DETECTION/EDGE_DETECTION.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+<Parameters>{ParametersSource}</Parameters>
+
+<SectionBreak />
+
+    
+
+[//]: # (Examples)
+
+## Examples
+
```
Comment (smahmed776) [inline docs/nodes/TRANSFORMERS/IMAGE_PROCESSING/EDGE_DETECTION/EDGE_DETECTION.md:28]: Examples should be imported here

**Review by smahmed776 — APPROVED** [review]
LGTM

**Pull request merged**